1.1.6/flake.lock

```diff
@@ -0,0 +1,27 @@
+{
+  "nodes": {
+    "nixpkgs": {
+      "locked": {
+        "lastModified": 1645047201,
+        "narHash": "sha256-NL8wLUIwAomUz9Aey4y0Eg96K/hdJ6owCqtWt044s84=",
+        "owner": "nixos",
+        "repo": "nixpkgs",
+        "rev": "3c2757d25d1ea5133fe8d207e995032b722b9e39",
+        "type": "github"
+      },
+      "original": {
+        "owner": "nixos",
+        "repo": "nixpkgs",
+        "rev": "3c2757d25d1ea5133fe8d207e995032b722b9e39",
+        "type": "github"
+      }
+    },
+    "root": {
+      "inputs": {
+        "nixpkgs": "nixpkgs"
+      }
+    }
+  },
+  "root": "root",
+  "version": 7
+}
```

Comment:
This `flake.lock` file is similar to a `package.json.lock` file in the JavaScript world, except we're locking all the inputs to our Nix flake. In this case, the only input is https://github.com/NixOS/nixpkgs.